tests/api2/test_smb_encryption.py

```diff
@@ -94,3 +105,27 @@ def test__smb_client_encrypt_required(smb_setup):
         # check share
         assert smb_status['share_connections'][0]['encryption']['cipher'] == 'AES-128-GCM'
         assert smb_status['share_connections'][0]['encryption']['degree'] == 'full'
+
+
+@pytest.mark.parametrize('enc_param', ('DESIRED', 'REQUIRED'))
+def test__smb_client_server_encrypt(smb_setup, enc_param):
+    with server_encryption(enc_param):
+        with smb_connection(
+            share=smb_setup['share']['name'],
+            username=SHAREUSER,
+            password=PASSWD,
+            encryption='DEFAULT'
+        ) as c:
+            # perform basic op to fully initialize SMB session
```

Comment:
Hmm.  Should the next line be the `c.ls('/')` ?

Reply:
Yes